modules/ROOT/nav.adoc

```diff
@@ -1,84 +1,79 @@
-* xref:con_services.adoc[Introducing Mobile Services]
-** xref:provisioning-services.adoc[Provisioning]
-** xref:integrating-services.adoc[Binding Services]
-* xref:apps.adoc[Integrating Mobile Apps]
-** xref:registering-a-mobile-app.adoc[Registering an App]
-** xref:binding-mobile-app-record-to-mobile-services.adoc[Binding an App to a Service]
-** xref:configuring-dev-env.adoc[Configuring a Dev Environment]
-* Using Mobile Services
+** xref:registering-a-mobile-app.adoc[Registering a Mobile App]
+** xref:configuring-dev-env.adoc[Dev Environment Setup]
+
+* xref:con_services.adoc[Mobile Services]
+
+// KEYCLOAK
+* xref:keycloak/index.adoc[{keycloak-service}]
+** xref:keycloak/provisioning.adoc[Provisioning]
+** xref:keycloak/binding.adoc[Binding]
+** xref:keycloak/coding.adoc[Coding]
+** xref:keycloak/authenticating.adoc[User Authentication]
+** xref:keycloak/retrieving-users-roles.adoc[Access Control]
+** xref:keycloak/sso.adoc[Single Sign On]
+** xref:metrics/monitoring-keycloak.adoc[Monitoring {keycloak-service}]
+
+
+
+// SECURITY
+* xref:security/index.adoc[{device-security-service}]
+** xref:security/proc_performing-device-checks.adoc[Device Trust Checks]
+** xref:security/proc_pinning-certificates.adoc[Certificate Pinning]
+
+
 // PUSH
-** xref:push/index.adoc[{unifiedpush-service}]
-*** xref:push/provisioning.adoc[Provisioning]
-*** xref:push/binding.adoc[Binding]
-*** xref:push/configuring-dev-env-push.adoc[Configuring a Dev Environment]
-*** xref:push/coding.adoc[Adding Code]
+* xref:push/index.adoc[{unifiedpush-service}]
+** xref:push/provisioning.adoc[Provisioning]
+** xref:push/binding.adoc[Binding]
+** xref:push/coding.adoc[Coding]
+** More...
 *** xref:push/proc_sending-a-push-notification.adoc[Sending a Push Notification]
-*** More...
-**** xref:push/admin-ui.adoc[{push-ui}]
-**** xref:push/handling-push-notifications.adoc[Handling Push Notifications]
-**** xref:push/proc_preparing-ups-for-use-with-ios.adoc[Apple Push Network]
-**** xref:push/google-setup.adoc[Firebase Cloud Messaging]
+*** xref:push/admin-ui.adoc[{push-ui}]
+*** xref:push/handling-push-notifications.adoc[Handling Push Notifications]
+*** xref:push/proc_preparing-ups-for-use-with-ios.adoc[Apple Push Network]
+*** xref:push/google-setup.adoc[Firebase Cloud Messaging]
 // METRICS
-** xref:metrics/index.adoc[{metrics-service}]
-*** xref:metrics/provisioning.adoc[Provisioning]
-*** Client metrics
-**** xref:metrics/binding.adoc[Binding]
-**** xref:metrics/configuring-dev-env-metrics.adoc[Configuring a Dev environment]
-**** xref:metrics/analysing-app-usage.adoc[Analysing App Usage]
-*** Service metrics
-**** xref:metrics/monitoring-keycloak.adoc[Monitoring {keycloak-service}]
+* xref:metrics/index.adoc[{metrics-service}]
+** xref:metrics/provisioning.adoc[Provisioning]
+** xref:metrics/binding.adoc[Binding]
+** xref:metrics/configuring-dev-env-metrics.adoc[Coding an App to use {metrics-service}]
+** xref:metrics/analysing-app-usage.adoc[Analysing Apps and Devices]
```

Comment:
`Analyzing Device Trust Checks` missing?

Reply:
we can add it under existing node for now, and deal with it later